docs/src/reference/types.md

```diff
@@ -901,3 +901,73 @@ You can access definitions from the module using
 >>>
 >>> #(-3)
 ```
+
+# Selector
+A filter for selecting elements within the document.
+
+You can construct a selector in the following ways:
+- you can call the `[selector]` function
+- you can use a `[position]`
+- you can use an element function
+- you can use a regex
+- you can use a `[<label>]`
+
+A selector is what you can use to query the document for certain types
+of elements. It can also be used to apply styling rules to element. You can
+combined multiple queries together using the methods on a selector.
+
+Selectors can also be passed to several of the built-in functions in typst to
+change their behaviour. Such as the [outline]($func/outline) where it can be
+use to change the type of items that are listed within the outline.
```

Comment:
```suggestion
Selectors can also be passed to several of Typst's built-in functions to configure 
their behaviour. One such example is the [outline]($func/outline) where it can be
use to change which elements are listed within the outline.
```